docs/development/workflow/get_devel_version.rst

```diff
@@ -15,7 +15,7 @@ Try the development version
     most of it is one-time setup. You can count on one hand the number of
     `git`_ commands you will need to regularly use to keep your local copy
     of `Astropy`_ up to date. If you find this taking more than an hour email
```

Comment:
this isn't your change, but since you're changing it, maybe "email" -> "send a message to", given that the link is to the web interface rather than a mailto (email link)